packer/vagrant/scripts/windows/msys2-install.bat

```diff
@@ -0,0 +1,5 @@
+if not exist "C:\Windows\Temp\msys2.exe" (
+powershell -Command "[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::TLS12; (New-Object System.Net.WebClient).DownloadFile('https://github.com/msys2/msys2-installer/releases/download/2020-06-02/msys2-x86_64-20200602.exe', 'C:\Windows\Temp\msys2.exe')" <NUL
```

Comment:
I noticed that this version of msys2 was flaky. I would sometimes run into this issue https://github.com/msys2/MSYS2-packages/issues/1429 explained more in this blog post https://www.msys2.org/news/#2020-06-29-new-packages. updating to the newest msys2 resolved it for me though. Not sure, if it's still an issue here

Reply:
It definitely is

Reply:
https://github.com/msys2/msys2-installer/releases/download/2020-07-20/msys2-x86_64-20200720.exe should be the most up to date one